lib/bootstrap_form/form_builder.rb

```diff
@@ -61,7 +61,8 @@ def initialize(object_name, object, template, options)
     end
 
     def file_field_with_bootstrap(name, options = {})
-      form_group_builder(name, options.reverse_merge(control_class: nil)) do
+      options[:control_class] ||= 'form-control-file'
```

Comment:
I would prefer `reverse_merge` here, because it creates a new hash instead of modifying the `options` in place.